mappings/servers/twerion/metadata.json

```diff
@@ -0,0 +1,59 @@
+{
+   "id":"twerion",
+   "name":"Twerion Network",
+   "description":"A network focused on PvP",
+   "addresses":[
+      "twerion.net",
+      "twerion.de",
+      "twerion.com",
+      "twerion.eu",
+      "twerion.org"
+   ],
+   "primaryAddress":"twerion.net",
+   "minecraftVersions":[
+      "1.8.*",
+      "1.9.*",
+      "1.10.*",
+      "1.11.*",
+      "1.12.*",
+      "1.16.*",
+      "1.17.*",
+      "1.18.*",
+      "1.19.*",
+      "1.20.*",
+      "1.21.*"
+   ],
+   "primaryMinecraftVersion":"1.8.8",
```

Comment:
The primary version needs to be one we already support

Reply:
Has been changed.